airbyte-integrations/connectors/source-stripe/source_stripe/schemas/invoices.json

```diff
@@ -20,6 +20,9 @@
     "description": {
       "type": ["null", "string"]
     },
+    "customer_tax_ids": {
+      "type": ["null", "array"]
```

Comment:
We should not use empty `properties`, if we expect an object.

If the `array` is expected there should be `items` populated instead.